apps/spruce/package.json

```diff
@@ -170,21 +163,9 @@
     "babel-preset-react-app": "^10.0.1",
     "camelcase": "^6.1.0",
     "cypress": "12.17.1",
-    "eslint": "8.19.0",
-    "eslint-config-airbnb": "19.0.4",
-    "eslint-config-prettier": "9.1.0",
+    "eslint": "8.31.0",
+    "eslint-config-evg": "*",
     "eslint-plugin-check-file": "2.6.2",
```

Comment:
Can we file a ticket to install `eslint-plugin-check-file` in Parsley so we can extract it out later?

Reply:
DEVPROD-6889